### DISCUSSION TITLE
Help with setup

### DISCUSSION THREAD

**Discussion by miran-hadziomerovic** [github/safe-settings#349] [Q&A]
Hi,

I am trying to setup the app to use within my organization, but I am unsuccessful. No apparent erros in the code is what is my issue.

I am trying to deploy the built docker image to an Azure webapp using azure/webapps-deploy@v2 action.

Here are my pre-release and release yamls. Am I doing anything obviously wrong? If you need more information, please let me know what can I provide.

I've converted the files to txt from yml so they can be uploaded
[create-pre-release-for-qna.txt](https://github.com/github/safe-settings/files/10154742/create-pre-release-for-qna.txt)
[create-release-for-qna.txt](https://github.com/github/safe-settings/files/10154743/create-release-for-qna.txt)

Any help would be greatly appreciated!

Thank you in advance.

**Comment by bheemvennapureddy** [comment]
i am deploying this manually on a ec2 on a localhost - created a deployment settings under .github created a include list a test app but doesn't seem to be working can i get some help ?

**Comment by svg153** [comment]
In case it helps you, you can look at my test organization...  https://github.com/svg153-org/admin Inside the apps/safe-setting-local/ folder I have a docker compose
